rules/anonymous-access-enabled/raw.rego

```diff
@@ -0,0 +1,30 @@
+package armo_builtins
+
+# Fails is rolebinding/clusterrolebinding gives permissions to anonymous user
+deny[msga] {
+    rolebindings := [rolebinding | rolebinding = input[_]; endswith(rolebinding.kind, "Binding")]
```

Comment:
Why do we need the endswith?

Reply:
I think is to be able to check for `RoleBinding` and `ClusterRoleBinding` at the same time.
Is it correct @YiscahLevySilas1 ?

Reply:
Right. We always do some validation of the input and this way we can check both resources in one deny block